src/plugins/sync-title/client.js

```diff
@@ -33,15 +33,22 @@ function installClientMutation(sync) {
 
 }
 
-module.exports = function clientSyncTitle(client) {
-
-	function sync(value) {
-		client.sendEvent('title', value);
-	}
+module.exports = function(syncPage) {
+	function clientSyncTitle(client) {
+
+		function sync(value) {
+			if (syncPage) {
+				client.sendEvent('title', value);
+			} else {
+				client.sendEvent('iframeTitleOnly', value);
```

Comment:
Instead of a separate event, `iframeTitleOnly` could also be passed as a boolean parameter on the existing `title` event.